application/src/main/resources/db/V2_Add_Warn_System.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE warns
```

Comment:
* why is the file called `V2`? there is already a `V2` isnt it? you have to increment this number
* `warns` is a bad table name. `user_warnings` maybe